packages/office-ui-fabric-react/src/components/GroupedList/GroupHeader.base.tsx

```diff
@@ -84,6 +81,15 @@ export class GroupHeaderBase extends React.Component<IGroupHeaderProps, IGroupHe
     if (!group) {
       return null;
     }
+
+    if (!expandButtonProps) {
+      expandButtonProps = {};
+    }
+    // set default aria label if expandButtonProps['aria-label'] is not defined.
+    if (!expandButtonProps['aria-label']) {
+      expandButtonProps['aria-label'] = group.isCollapsed ? 'expand list' : 'collapse list';
```

Comment:
If we're doing this, should it be 'expand group' and 'collapse group'?

Reply:
If we change this, could we also update the doc comment in the GroupHeader.types.ts file?

Maybe it could say something like this:

```tsx
/** 
* Native props for the GroupHeader expand and collapse button 
* 'aria-label' defaults to 'expand list' if `group.isCollapsed` is true and 'collapse list' if `group.isCollapsed' is false
*/
  expandButtonProps?: React.HTMLAttributes<HTMLButtonElement>;
```

@KevinTCoughlin could you take another look at these changes?

Also side note - while we're here, this also brings up the question of defaulting to en-us strings in our controls. Back in August of last year we were talking about a common localization model and how we could all benefit from it https://github.com/OfficeDev/office-ui-fabric-react/issues/5047#issuecomment-412234130. Thanks for surfacing this discussion again! I'll loop back with the team to see if progress has been made there. :)

Reply:
I've updated the aria-labels and comments as requested